src/features/translation/ui/translation.tsx

```
@@ -131,25 +64,24 @@ export const Translation = ({
+          onPaste={onPaste}
+          onKeyDown={onKeyDown}
+          placeholder="Enter your paragraph here"
+          height="3xs"
+          mb="2"
+          ref={(e) => {
+            register("input").ref(e);
+            inputRef.current = e;
+          }}
+        />
+        <FormErrorMessage mb="2">
+          {errors.input && errors.input.message}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)